### ISSUE TITLE
falcon_unzip breaks down when running blasr

### ISSUE THREAD

**Issue by yilunhuangyue** [PacificBiosciences/FALCON_unzip#77]
hello, I met an error when running falcon_unzip, It seems the pipe stopped when running blasr, there was just tmp_aln.bam.tmp in /3-unzip/0-phasing/000000F/blasr, the last lines of stderr shows as follows:

[INFO] 2017-03-30T11:33:05 [blasr] started.
Warning: When attempting to select equivalently scoring reads at random 
the bestn parameter should be greater than one.
touch /home02/wuguizhi/JLX/Falcon_test3/3-unzip/0-phasing/000000F/blasr/aln_000000F_done.exit
++ touch /home02/wuguizhi/JLX/Falcon_test3/3-unzip/0-phasing/000000F/blasr/aln_000000F_done.exit

the blasr version is 5.3.f8bfa9c. 

I don not know what is the problem. 

Thanks a lot for any help.

**Comment by pb-cdunn** [comment]
I don't think this is a version issue. We'll need your help to debug this. You can `cd` into the task directory and repeat just this task. There are shell scripts which show sub-shells. Hopefully you can learn something from there.

**Comment by yilunhuangyue** [comment]
Sorry for the late reply. It seems that the error was happened because I did not import the libraries of blasr. 
(error while loading shared libraries: libpbihdf.so: cannot open shared object file: No such file or directory).
The blasr I used was installed by pitchfork, after export the libraries, the blasr is running now.
I will send the message if I meet other errors.
Thanks again for you reply.
